tikzplotlib/_path.py

```diff
@@ -22,8 +23,10 @@ def draw_path(data, path, draw_options=None, simplify=None):
     ):
         return data, "", None, False
 
+    x_is_date = isinstance(data["current mpl axes obj"].xaxis.converter, DateConverter)
```

Comment:
If there's a better way to detect the presence of a datetime axis, please edit accordingly